onnx/test/compose_test.py

```diff
@@ -0,0 +1,216 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+
+from typing import Text, List
+
+import onnx
+import onnx.version_converter
+from onnx import helper, TensorProto, GraphProto, ValueInfoProto
+
+
+def _create_tensor(name, dtype=TensorProto.FLOAT, shape=[1, 2]):  # type: ignore
+    return helper.make_tensor_value_info(name, TensorProto.FLOAT, shape)
+
+
+class TestComposeFunctions(unittest.TestCase):
+    def test_merge(self):  # type: () -> None
+        A0 = _create_tensor("A0")
+        A1 = _create_tensor("A1")
+        B00 = _create_tensor("B00")
+        B10 = _create_tensor("B10")
+        B20 = _create_tensor("B20")
+
+        L0_0 = helper.make_node("Add", ["A0", "A1"], ["B00"], "L0_0")
+        L0_1 = helper.make_node("Sub", ["A0", "A1"], ["B10"], "L0_1")
+        L0_2 = helper.make_node("Mul", ["A0", "A1"], ["B20"], "L0_2")
+
+        g1 = helper.make_graph(
+            [L0_0, L0_1, L0_2],
+            "test1",
+            [A0, A1],
+            [B00, B10, B20])
+
+        B01 = _create_tensor("B01")
+        B11 = _create_tensor("B11")
+        B21 = _create_tensor("B21")
+        D0 = _create_tensor("D0")
+
+        L1_0 = helper.make_node("Add", ["B01", "B11"], ["C0"], "L1_0")
+        L1_1 = helper.make_node("Sub", ["B11", "B21"], ["C1"], "L1_1")
+        L2_0 = helper.make_node("Mul", ["C0", "C1"], ["D0"], "L2_0")
+
+        g2 = helper.make_graph(
```

Comment:
this is good but do check onnx parser... this utility makes it very easy to create onnx models for test purposes. Example: https://github.com/onnx/onnx/blob/master/onnx/test/parser_test.py#L27

Reply:
Much easier with onxx.parser. Thanks for the suggestion. I updated the test code.